lib/templates/test-body-footer.html

```diff
@@ -47,6 +47,14 @@
       return;
     }
 
+    if (typeof data === 'string') {
```

Comment:
I guess I still want to understand why `data` would be a string. 🤔

Reply:
@xcambar - Have you looked into this?

Reply:
Not thoroughly, but an initial look did not exclude that the behavior originates in `istanbul-lib-coverage`, unfortunately.